src/test/scala/io/iohk/ethereum/extvm/VMServerSpec.scala

```diff
@@ -0,0 +1,117 @@
+package io.iohk.ethereum.extvm
+
+import akka.util.ByteString
+import com.trueaccord.scalapb.GeneratedMessageCompanion
+import io.iohk.ethereum.domain.{Account, Address}
+import io.iohk.ethereum.extvm.msg.{CallContext, VMQuery}
+import io.iohk.ethereum.utils.BlockchainConfig
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{FlatSpec, Matchers}
+
+class VMServerSpec extends FlatSpec with Matchers with MockFactory {
```

Comment:
I realise queries are tested within `VMClientSpec`, should they be tested here as well? 🤔